docs/.vitepress/theme/components/Home.vue

```diff
@@ -165,14 +166,70 @@ const subscribeToNewsletter = async () => {
   </video>
 </div>
 
+<div id="whykitops" class="mt-32 md:mt-40 xl:mt-60 px-6 md:px-12 content-container">
+  <h2 class="text-center">Why Kit<span class="font-heading font-extralight">?</span></h2>
+
+  <div class="mt-10 md:mt-14 xl:mt-22 grid grid-cols-1 md:grid-cols-2 gap-4 md:gap-x-4 md:gap-y-[4.5rem] xl:gap-22 max-w-[47.5rem] mx-auto">
+    <div class="h4 !text-gold">Model handoffs are hard.</div>
+    <div class="p2 space-y-4">
+      <p>Moving a model from a Jupyter notebook to an ML tool or development server, then to a production server like Kubernetes is difficult because each tool uses its own packaging mechanism, and requires engineers to repackage the model multiple times. This slows down development and introduces risk.</p>
```

Comment:
This first sentence reads a little clumsy

Reply:
Let's keep for now, I'll work on smoothing it out on the next iteration.